Probesys/bileto
# tec: Remove sub-organizations

## Related issue(s)



Relates to https://github.com/Probesys/bileto/issues/516
Fixes https://github.com/Probesys/bileto/issues/496

## How to test manually



1. create an organization
2. add a ticket in the organization
3. add a contract in the organization
4. delete the organization
5. check that all the operations work

## Checklist
